src/Serializable.swift

```diff
@@ -92,6 +92,8 @@ public class Serializable: NSObject {
                 setValue(propertiesDictionary, value: NSNumber(unsignedInt: propValue), forKey: propName!)
             } else if let propValue: UInt64 = propValue as? UInt64 {
                 setValue(propertiesDictionary, value: NSNumber(unsignedLongLong: propValue), forKey: propName!)
+            } else if isEnum(propValue){
```

Comment:
Opening Brace Spacing Violation: Opening braces should be preceded by a single space and on the same line as the declaration. (opening_brace)